packages/plugin-github/sources/githubUtils.ts

```diff
@@ -35,7 +35,12 @@ export function parseGithubUrl(urlStr: string): ParsedGithubUrl {
 
   let [, auth, username, reponame, treeish = `master`] = match;
 
-  treeish = treeish.replace(/[^:]*:/, ``);
+  // Replace Treeish Protocol
+  treeish = treeish
+    // New-style: "commit=abcdef" (without the extras)
+    .replace(/[^=]*=/, ``)
+    // Old-style: "commit:abcdef"
+    .replace(/[^:]*:/, ``);
```

Comment:
It would be sounder to use `querystring.parse` and explicitly extract `commit` (there can be no other field, since the normalization pass turned tags and semvers into commit hashes).

Reply:
Done. I forgot that all URLs are normalized by the `GitResolver`. 😅